### ISSUE TITLE
Documentation from tutorial does not work

### ISSUE THREAD

**Issue by igorshubovych** [unmock/unmock-js#93]
The last example in this tutorial does not work
https://www.unmock.io/hello.html

```js
test("setting a value for endpoint", async () => {
  unmock.states().hello({ hello: "world" });
  var res = await axios.get("https://api.unmock.io");
  expect(res.data).toEqual({ hello: "world " });
});
```

First, there is a space on the last line:
```js
expect(res.data).toEqual({ hello: "world " }); // Unnessary space after world
```

Then, it does not overwrite the response, the test fails, because the response is still "foo", not "world".

**Comment by idantene** [comment]
Hey @igorshubovych,
Thank you for reporting this! I seems like this is an old part of the tutorial. Are you following the code [here](https://www.unmock.io/hello.html#creating-our-first-test)?

Could you please include the full output of your test results?

**Comment by idantene** [comment]
Hey - thank you for the information. We've managed to track down the issue and it should be fixed soon (version 0.1.3).
